lib/impl/testing_utils.dart

```diff
@@ -0,0 +1,36 @@
+
+/// Return the list of possible test paths. The given [path] must be in the
+/// `lib/` directory.
+List<String> getPossibleTestPaths(String path, { String separator: '/' }) {
```

Comment:
Perhaps change
```
{ String separator: '/' } {
```
to
```
{ String separator } {
  separator ??= fs.separator;
```
?

That way most (all?) callers would not have to remember to pass in the separator and we're less likely to have a Windows specific bug.